platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/file/FileHandler.java

```diff
@@ -77,6 +80,19 @@ public FileHandler(
         } catch (IOException e) {
             throw new IOException("Could not create FileHandler", e);
         }
+        BaseExecutorFactory.getInstance()
+                .scheduleAtFixedRate(
+                        () -> {
+                            try {
+                                outputStream.flush();
+                            } catch (IOException e) {
+                                EMERGENCY_LOGGER.log(
+                                        Level.WARN, "Unable to flush " + handlerName + " nothing to do, will retry", e);
```

Comment:
I do not get the message.

Reply:
Let me rethink the message
Given that it is a periodic operation that is automatically invoked, I thought it was best to try to communicate that it was not an end-of-the-world failure...